docs/templates/Partial Templates/Geometry/Curve Segment Geometry/Clothoid Transition Segment/README.md

```diff
@@ -1,7 +1,11 @@
 Clothoid Transition Segment
 ===========================
 
-A clothoid segment is based on the IfcClothoid where the value for the clothoid constant is specified as √L and L is the length measured from the inflection point.
+A clothoid segment is based on the IfcClothoid where the value for the clothoid constant _A_, known as _flatness_ or _homothetic parameter_ of the clothoid, is specified as:
```

Comment:
We have a clickable link here 
![afbeelding](https://github.com/buildingSMART/IFC4.3.x-development/assets/1096535/cef2d4f6-6691-4de9-829b-61105991689c)
 Do we really need to duplicate the text?